thunder/tests/test_ops.py

```diff
@@ -251,3 +251,27 @@ def fn(a):
     out = jf(a)
     assert_close(a, a_ref)
     assert_close(out, out_ref)
+
+
+@requiresCUDA
+def test_exponential():
+    def fn(a):
+        return a.exponential_(1)
+
+    size = 1000
+    seed = 1234
+
+    # on cpu, aten.exponential_ is not decomposed to ops used in Thunder exponential_.
```

Comment:
What about adding a direct lowering to the PyTorch executor?

How different is PyTorch's Tensor.exponential_ on CPU and CUDA? Are they so different that we should refuse to execute exponential on the CPU for now (and file an issue to follow-up)?

Reply:
Pardon the naive question, but how would that work given that on CPU, PyTorch lowers to inplace `exponential_` ATen kernel and on GPU, it lowers to the decomposition shown here, and we don't support inplace ops in thunder atm? As in how would we handle the cpu case with just PyTorch. Thanks.

Reply:
We can create a `prim.exponential` and then have that call `torch.ops.aten.exponential`, which will - by definition - do the same thing as PyTorch, and instead of decomposing lower to that prim.

However, when we don't require NVFuser to give us the same random numbers as PyTorch, why would we require this. In practice, we'll use NVFuser's random numbers.